index.js

```diff
@@ -184,11 +185,17 @@ function TorrentGraph (root) {
                 link.style('opacity', 0.3)
 
                 focus = false
+                if (undefined !== onSelect) {
```

Comment:
This looks better when written as `onSelect !== undefined`. That's how we do all our equality tests in the codebase.